### PR TITLE
Migrated Batch 7 of Thermal Conductivity

### PR DESCRIPTION
**Pull request by qur786**
@maneetgoyal sir please look at the data of silicon Tetra Fluoride

### CODE REVIEW

File: src/data/thermal-conductivity-constants.ts [hunk 1]
```diff
@@ -2774,6 +2774,726 @@ export const DIPPRThermalConductivityDictionary: Record<
...
+    minimumTemperature: 223.15,
+    maximumTemperature: 318.69,
+    thermalConductivityAtMinimumTemperature: 0.1072,
+    thermalConductivityAtMaximumTemperature: 0.0442,
+  },
+
+  SulfurTrioxide: {
+    name: 'Sulfur trioxide',
+    formula: 'O3S',
+    CAS: '09-11-7446',
```
Comment (saikat-samanta) [inline src/data/thermal-conductivity-constants.ts:3474]: 7446-11-9

**Review by saikat-samanta — APPROVED** [review]

File: src/data/thermal-conductivity-constants.ts [hunk 2]
```diff
@@ -2774,6 +2774,726 @@ export const DIPPRThermalConductivityDictionary: Record<
...
+  },
+
+  Pentane: {
+    name: 'Pentane',
+    formula: 'C5H12',
+    CAS: '109-66-0',
+    molecularWeight: 72.14878,
+    C1: 0.2537,
+    C2: -0.000576,
+    C3: 0.000000344,
```
Comment (maneetgoyal) [inline src/data/thermal-conductivity-constants.ts:2994]: You can use exponential notation for better readability.

File: src/data/thermal-conductivity-constants.ts [hunk 3]
```diff
@@ -2774,6 +2774,726 @@ export const DIPPRThermalConductivityDictionary: Record<
...
+    name: 'Silicon tetrafluoride',
+    formula: 'F4Si',
+    CAS: '7783-61-1',
+    molecularWeight: 104.07911,
+    C1: NaN,
+    C2: NaN,
+    minimumTemperature: NaN,
+    maximumTemperature: NaN,
+    thermalConductivityAtMinimumTemperature: NaN,
+    thermalConductivityAtMaximumTemperature: NaN,
```
Comment (maneetgoyal) [inline src/data/thermal-conductivity-constants.ts:3416]:
```suggestion
```

- [ ] Update the interface as needed
- [ ] Add a unit test for this compound

File: src/data/thermal-conductivity-constants.ts [hunk 4]
```diff
@@ -2774,6 +2774,726 @@ export const DIPPRThermalConductivityDictionary: Record<
...
+    minimumTemperature: 460.85,
+    maximumTemperature: 591,
+    thermalConductivityAtMinimumTemperature: 0.1653,
+    thermalConductivityAtMaximumTemperature: 0.1351,
+  },
+
+  SulfurDioxide: {
+    name: 'Sulfur dioxide',
+    formula: 'O2S',
+    CAS: '05-09-7446',
```
Comment (maneetgoyal) [inline src/data/thermal-conductivity-constants.ts:3448]:
```suggestion
    CAS: '7446-09-05',
```

**Review by maneetgoyal — COMMENTED** [review]
Some updates needed

**Review by maneetgoyal — APPROVED** [review]
👍

**Pull request merged**